src/display/api.js

```diff
@@ -94,6 +88,15 @@ if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('GENERIC')) {
     });
   }) : null;
 }
+if (typeof PDFJSDev !== 'undefined' &&
+    PDFJSDev.test('PRODUCTION && !(MOZCENTRAL || FIREFOX)') &&
+    !fallbackWorkerSrc && typeof document !== 'undefined') {
+  const pdfjsFilePath = document.currentScript && document.currentScript.src;
```

Comment:
I initially considered inlining the [`document.currentScript` polyfill](https://github.com/mozilla/pdf.js/blob/master/src/shared/compatibility.js#L57-L74) here, such that it could be removed since `document.currentScript` is used *once*, but that might not be a desired change!?

Reply:
Indeed, let's keep it in `compatibility.js` even if it's only used once because I think it's better to consolidate all polyfills in one place as much as possible so we can easily go through them and remove them when they are no longer relevant.